### PR TITLE
SEO-173276-WPF-Image-Changes

### PR DESCRIPTION
**Pull request by Scholar-jaruha**
Hi @MallikaRK,
Please find changes made under image section.
Regards,
Scholar.

### CODE REVIEW

File: wpf/Radial-Gauge/Header.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ this.Content = sfCircularGauge;
 
 {% endtabs %}
 
-![CircularGauge Header](Header_images/Header_img1.png)
+![CircularGauge Header in WPF Radial Gauge](header_images/wpf-radial-gauge-circular-gauge-header.png)
```
Comment (MallikaRK) [inline wpf/Radial-Gauge/Header.md:89]: why you mentioned two control names here. 
circular gauge/radial gauge.
Use any one of the control name. Check and update in all places.

File: wpf/Radial-Gauge/Header.md [hunk 2]
```diff
@@ -261,7 +261,7 @@ The [`GaugeHeaderPosition`](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xam
 
 {% endtabs %}
 
-![CircularGauge header with custom position](Header_images/Header_img3.png)
+![CircularGauge header with custom position in WPF Radial Gauge](header_images/wpf-radial-gaugecircular-gauge-header-custon-position.png)
```
Comment (MallikaRK) [inline wpf/Radial-Gauge/Header.md:264]: recheck image name.

File: wpf/Radial-Gauge/Header.md [hunk 3]
```diff
@@ -345,7 +345,7 @@ You can customize the header’s text by using the `FontFamily`, `FontStyle`, `F
 
 {% endtabs %}
 
-![CircularGauge Header with customization](Header_images/Header_img4.png)
+![CircularGauge Header with customization in WPF Radial Gauge](header_images/wpf-radial-gauge-circular-gauge-header-customization.png)
```
Comment (MallikaRK) [inline wpf/Radial-Gauge/Header.md:348]: Use any one of the control name. 
Check and update in all places.

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

File: wpf/Radial-Gauge/Getting-Started.md [hunk 4]
```diff
@@ -18,7 +18,7 @@ You can add gauge reference using one of the following methods:
 
 Syncfusion WPF components are available in [`nuget.org`](https://www.nuget.org/). To add gauge to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.SfGauge.WPF](https://www.nuget.org/packages/Syncfusion.SfGauge.WPF), and then install it.
 
-![Adding gauge reference from NuGet](Getting-Started_images/Adding gauge reference.png)
+![Adding gauge reference from NuGet in WPF Radial Gauge](getting-started_images/wpf-radial-gauge-adding-gauge-reference.png)
```
[the remark this answers is not in the document]
Comment (Scholar-jaruha) [reply wpf/Radial-Gauge/Getting-Started.md:21]: Image name changed

File: wpf/Radial-Gauge/Header.md (hunk 1 above)
Comment (Scholar-jaruha) [reply wpf/Radial-Gauge/Header.md:89]: Changed

File: wpf/Radial-Gauge/Header.md (hunk 2 above)
Comment (Scholar-jaruha) [reply wpf/Radial-Gauge/Header.md:264]: changes made

File: wpf/Radial-Gauge/Header.md (hunk 3 above)
Comment (Scholar-jaruha) [reply wpf/Radial-Gauge/Header.md:348]: updated

**Review by MallikaRK — APPROVED** [review]
Approved

**Comment by Aishwarya-Ganesan** [comment]
Rebuild

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**